placideirandora/epic-mail
# #164150032 built inbox content

#### What does this PR do?
It adds inbox content to the view inbox dashboard menu.

#### Description of Task to be completed?
1. Adds a link of inbox content to the view inbox dasboard menu.
2. When the view inbox dashboard menu is clicked, a list of received emails is displayed in its appropriate page section.

#### How should this be manually tested?
Click and navigate to the view inbox content and then you will see a list of all received emails.
